src/storage/filesystem-adapter.test.ts

```
@@ -0,0 +1,150 @@
+    it('should handle clear errors gracefully', async () => {
+      // Create a directory at the file path to cause a delete error
+      await fs.mkdir(testPath, { recursive: true });
+      
+      // Create a file inside the directory to make it non-empty
+      const filePath = path.join(testPath, 'file.txt');
+      await fs.writeFile(filePath, 'test');
+
+      await expect(adapter.clear()).rejects.toThrow('Failed to clear cache data from filesystem');
+    });
+  });
+}); 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)